src/main/java/com/google/sps/data/Project.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.data;
+
+/** Represents a project. */
+public class Project {
+
+  public String name;
+  public String id;
+  public String projectNumber;
```

Comment:
Should this be a "ProjectIdentification"?

Reply:
I can wrap it in a projectIdentification object